classes/kernel-cve-check.bbclass

```diff
@@ -130,17 +130,29 @@ python kernel_cve_check () {
     cmd = "SSL_CERT_FILE='%s' %s scripts/report_affected.py %s --remote-name cip:%s --git-repo %s %s" % (cert_path, python_path, opt_ignore, remote_repo_name, kernel_path, linux_cip_ver)
     output = runfetchcmd(cmd, d, workdir=cip_kernel_sec_path)
 
-    affect_cves = output.split(":")[2].strip().split()
+    tmp_affected_cves = output.split(":")[2].strip().split()
 
     whitelist = []
     for cve in get_issue_list(cip_kernel_sec_path):
-        if cve not in affect_cves:
+        if cve not in tmp_affected_cves:
             whitelist.append(cve)
 
     bb.debug(2, "Whitelisted by cip-kernel-sec:\n    %s" % "\n    ".join(whitelist))
     d.appendVar("CVE_CHECK_WHITELIST", ' ' + ' '.join(whitelist))
 
-    d.setVar("CIP_KERNEL_SEC_REPORTED_CVES", " ".join(affect_cves))
+    allowlist_str = d.getVar("CVE_CHECK_WHITELIST")
+    allowlist = allowlist_str.split(" ")
+
+    patched_cves = []
+    affected_cves = []
+    for cve in tmp_affected_cves:
+        if not cve in allowlist:
+            affected_cves.append(cve)
+        else:
+            patched_cves.append(cve)
```

Comment:
I think it would be enough to just remove the allow(safe)listed CVEs from the affected_cves. Is it necessary to make the patched_cves which is going to be marked in check_cves() later?

I assume the patched_cves will necessary to show "Patched" CVEs detected by cip-kernel-sec but not by NVD in the future, but not now.